acceptance_tests/features/pages/sign_in_internal.py

```diff
@@ -1,9 +1,12 @@
+import time
+
 from acceptance_tests import browser
 from config import Config
 
 
 def go_to():
     browser.visit(Config.RESPONSE_OPERATIONS_UI + '/sign-in')
+    time.sleep(2)
```

Comment:
What does this have to do with the PR?

Reply:
For some reason, the acceptance tests were failing without this. Take it out and try it yourself.